images/build/ruby/2.7/Dockerfile

```diff
@@ -0,0 +1,3 @@
+# Build ruby 2.7
+FROM ruby-build-prereqs AS ruby27-build
```

Comment:
For runtime dockerfiles for ruby, you can check with Sanchit to get their existing Ruby related dockerfiles.